zhmcclient/_session.py

```diff
@@ -935,7 +935,7 @@ class Job(object):
     completion.
     """
 
-    def __init__(self, session, uri):
+    def __init__(self, session, uri, oper_method, oper_uri):
```

Comment:
I would like to see 'op' instead of 'oper' as an abbreviation for operation.
Why do you not use 'op_http_method' and 'op_uri'